python/graphstorm/gconstruct/transform.py

```diff
@@ -372,7 +431,7 @@ def after_merge_transform(self, feats):
         feats = np.clip(feats, -1 + self._epsilon, 1 - self._epsilon)
         feats = erfinv(feats)
 
-        feats = _feat_astype(feats, self._out_dtype)
+        self.as_out_dtype(feats)
```

Comment:
do you need to call it here?

Reply:
For RankGaussTransform, yes. 
Its feats is transformed in after_merge_transform(), which is after __call__() call.